### DISCUSSION TITLE
Question about the use of InterReactiveTransferFoam

### DISCUSSION THREAD

**Discussion by Santhosh91000** [GeoChemFoam/GeoChemFoam#19] [Q&A] [answered]
Hello all,

I have a general question about the use of InterReactiveTRansferFoam. Indeed, when we derive the equations implemented in the solver, there is the use of the Henry constant.

My question is that it works nicely for CO2 emulsion in water but what about liquid or supercritical CO2 ? Does the solver will still accurately or appropriately describe the reality ?

Thanks for the time and any hints or help is appreciated!

Sincerely,
Santhosh

**Comment by hannahmenke** [comment, marked as answer]
As long as the correct values are provided in the input files and chemical database, then liquid and SC-CO2 are no problem.

**Reply by Santhosh91000** [reply]
Hello Hannah,

Thanks for the reply!
